specification/gedcom.md

```diff
@@ -393,58 +393,35 @@ For FamilySearch GEDCOM 7.0, this is the `GEDC` structure described in [Chapter
 A header should contain an extension schema structure with tag `SCHMA`
 as described in [Extensions].
 
-##  Extensions
 
-A **standard structure** is a structure whose type, tag, meaning, superstructure, and cardinality within the superstructure are described in this document. This includes records such as `INDI` and substructures such as `INDI`.`NAME`.
+##  Extensions
 
-Two forms of **extension structures** are permitted:
+The structures documented in this specification are called **standard structures**.
+Each standard structure has
 
-- A **tagged extension structure** is a structure whose tag matches production `extTag`. Tagged extension structures may appear as records or substructures of any other structure.
-- An **extended-use standard structure** is a structure whose type, tag, and meaning are defined in this document and whose superstructure is a tagged extension structure.
+- A structure type, identified by a URI
+- A set of locations where it may appear (as a record or as a substructure of a set of superstructure types), with a permitted cardinality in each location
+- A set of substructures it may contain, with a permitted cardinality for each
+- A payload type; if an enumeration, the set of allowed enumerated values
+- A meaning
```

Comment:
```suggestion
- A structure type, identified by a URI;
- A set of locations where it may appear (as a record or as a substructure of a set of superstructure types), with a permitted cardinality in each location;
- A set of substructures it may contain, with a permitted cardinality for each;
- A payload type; if an enumeration, the set of allowed enumerated values;
- A meaning.
```
end lines in punctuation consistent with lines 416-419